website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-setting-up-bigquery-oauth.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Setting up BigQuery OAuth"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
+
+:::
+
+dbt Cloud supports [OAuth authentication](https://cloud.google.com/bigquery/docs/authentication) with BigQuery. When BigQuery OAuth is enabled, users can interact with the BigQuery warehouse as individual users, rather than leveraging a shared service level authentication. 
+
+:::info Some Pre-Work Required
+
+Before setting up a Client ID & Secret, you'll have to have your existing BigQuery Settings in order. We recommend using a Service Account JSON file, and have a walkthrough for that [here](tutorial/setting-up/#generate-bigquery-credentials) - you will also need to set up an [OAuth Consent Screen](https://support.google.com/cloud/answer/6158849) if you haven't already!
```

Comment:
```suggestion
Before setting up a Client ID & Secret, you'll have to have your existing BigQuery Settings in order. We recommend using a Service Account JSON file, and have a walkthrough for that [here](tutorial/setting-up#generate-bigquery-credentials) - you will also need to set up an [OAuth Consent Screen](https://support.google.com/cloud/answer/6158849) if you haven't already!
```

Reply:
Hmmm the relative link isn't working! It directs to `http://localhost:3000/docs/dbt-cloud/cloud-configuring-dbt-cloud/tutorial/setting-up/#generate-bigquery-credentials` which is clearly not correct 😓 

I'll change it to 'https://docs.getdbt.com/tutorial/setting-up#create-a-bigquery-project'